src/index.js

```diff
@@ -75,23 +80,24 @@ export default class Loader extends EventEmitter {
         const childLoader = new Loader(vastAdTagURI, null, this)
         return childLoader.load()
       })
-      .then((children) => {
-        return [vast, ...children]
-      })
+      .then((children) => ([vast, ...children]))
   }
 
   _fetchUri () {
     const fetching = fetch(this._uri, this._fetchOptions)
     const timingOut = this._createTimeouter(fetching)
+    let headers
     return Promise.race([fetching, timingOut])
       .then((response) => {
         timingOut.cancel()
         if (!response.ok) {
           // TODO Convert response to HTTPError
           throw new VASTLoaderError(900, response)
         }
+        headers = response.headers
```

Comment:
why not pass an anonymous object with `{headers, body}`? You're going to construct it in the `then` handler anyway? Then you don't need the outer variable.